compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/CompilerUtils.kt

```diff
@@ -102,6 +102,13 @@ public fun KotlinType.classDescriptor(): ClassDescriptor {
     )
 }
 
+@ExperimentalAnvilApi
+public fun AnnotationDescriptor.boundTypeOrNull(module: ModuleDescriptor): ClassDescriptor? {
```

Comment:
As far as I can tell this is ONLY used in the ClassDescriptor. Then we don't need this extra utility and it should be inlined. If it's used elsewhere, then this utility should be deprecated.